src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateBuildOptionsRule.cs

```diff
@@ -114,11 +114,22 @@ private AddPropertyTransform<CommonCompilerOptions>[] KeyFileTransforms
             "**/*.cs"
         };
 
+        private static bool IsPlainFileName(string fileName)
+        {
+            return !fileName.Contains('/') && !fileName.Contains('\\');
```

Comment:
I am a bit afraid of that because there may be globings that do not start with that but that still end-up outside of the repo.